### PR TITLE
Topics & namespaces & Digia IRIS eye icons into transitlog flowchart

### PR DESCRIPTION
**Pull request by Divinare**
TODO later: flowchart of Transitlog HFP pipeline (there is a card at backlog for this)

Closes https://github.com/HSLdevcom/transitdata/issues/9

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,10 +22,17 @@ Overall system requirements for running the system are:
 ## System Architecture & Components
 
 ![Alt text](transitdata_data_flow_drawio.png?raw=true "Transitdata System Architecture")
+![Alt text](transitdata_data_flow_drawio_details.png?raw=true "Transitdata System Architecture - a more detailed version")
```
Comment (kenkus-futurice) [inline README.md]: Are these supposed to show `omm-doi-roi-transitdata.png` and not `transitdata_data_flow_drawio.png` which was removed?

File: README.md [hunk 2]
```diff
@@ -22,10 +22,17 @@ Overall system requirements for running the system are:
...
 ![Alt text](transitlog_hfp_data_flow_drawio.png?raw=true "Transitlog System Architecture")
 
 Components are stored in their own Github Repositories:
+
+#### Common components
+
 - [transitdata-common](https://github.com/HSLdevcom/transitdata-common) contains generic components and shared constants.
```
Comment (kenkus-futurice) [inline README.md]: Strictly speaking `transitdata-common` is not a component like the others but a dependency.

File: README.md [hunk 3]
```diff
@@ -22,10 +22,17 @@ Overall system requirements for running the system are:
...
+
+#### Transitdata components
```
Comment (kenkus-futurice) [inline README.md]: Nice categories.

File: README.md (hunk 1 above)
Comment (kenkus-futurice) [reply README.md]: In the `omm-doi-roi-transitdata.png` image perhaps the namespace for "internal messages" should be "internal-messages" with the dash.

File: README.md (hunk 2 above)
Comment (Divinare) [reply README.md]: Changed it now as common depencencies

File: README.md (hunk 1 above)
Comment (Divinare) [reply README.md]: Fixes done

**Comment by Divinare** [comment]
Review fixes done

**Pull request merged**